imports/api/HistoryUtils.js

```diff
@@ -0,0 +1,45 @@
+import { Constants } from './Constants';
+import { EloCalculator } from '../api/EloCalculator.js';
+import { JapaneseHands } from './GameDatabases.js';
+import { Players } from './Players.js';
+
+export let HistoryUtils = {
+    regenerateJapaneseElo() {
```

Comment:
I don't think this is the right way to implement this.  All statistics should be regenerated at the same time, not just ELO.